clients/src/main/resources/common/message/ConsumerGroupDescribeResponse.json

```diff
@@ -69,7 +70,9 @@
             { "name": "Assignment", "type": "Assignment", "versions": "0+",
               "about": "The current assignment." },
             { "name": "TargetAssignment", "type": "Assignment", "versions": "0+",
-              "about": "The target assignment." }
+              "about": "The target assignment." },
+            { "name": "IsClassic", "type": "int8", "versions": "1+", "default": "-1", "ignorable": true,
```

Comment:
Should we consider renaming it back to "memberType" after the type is changed to `int8`?

Reply:
Hi @chia7712, I prefer to use `IsClassic` because it's more Intuitive. Also, we don't plan to expand the field as `protocol`, so I think we can keep it as `IsClassic`.

Reply:
>  I prefer to use IsClassic because it's more Intuitive.

I agree that using `IsClassic=false` and `IsClassic=true` would be more intuitive. However, currently, `IsClassic` can take the values `-1`, `0`, and `1`. These "magic numbers" are unclear without referring to the documentation.

Additionally, the codebase uses these magic numbers inconsistently across different classes. For example, several classes such as `ScramMechanism`, `EndpointType`, `ResourceType`, and `AclOperation` use `0` to represent "unknown" or "none". `MemberState` use `127` for unknown. 

In summary, it seems to me the name `IsClassic` is no longer intuitive after adopting the int8 type.

Reply:
Hi @chia7712, thanks for the suggestion. Yes, it's good to follow naming convention. I update both PR and KIP.

Reply:
I agree with @chia7712. Small nit. I wonder if we should now use 0 for classic member and 1 for consumer member as classic is the older one.

Reply:
>  I wonder if we should now use 0 for classic member and 1 for consumer member as classic is the older one.

+1, that order is intuitive to me

Reply:
Thanks for the suggestion. Updated both PR and KIP.